src/System.Runtime.Analyzers/UnitTests/SpecifyCultureInfoTests.cs

```diff
@@ -9,12 +10,320 @@ public class SpecifyCultureInfoTests : DiagnosticAnalyzerTestBase
     {
         protected override DiagnosticAnalyzer GetBasicDiagnosticAnalyzer()
         {
-            return new BasicSpecifyCultureInfoAnalyzer();
+            return new SpecifyCultureInfoAnalyzer();
         }
 
         protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer()
         {
-            return new CSharpSpecifyCultureInfoAnalyzer();
+            return new SpecifyCultureInfoAnalyzer();
+        }
+
+        [Fact]
+        public void CA1304_CSharp()
```

Comment:
I find it really hard to review a test that's structured like this. For each line that might or might not have a diagnostic, I have to determine the line number of the candidate line, then scroll down 150 lines to see whether or not you reported a diagnostic for that line.

I'll defer reviewing this for now.